templates/register.html

```diff
@@ -9,5 +9,43 @@ <h1>Sign Up</h1>
     <input type="password" name="password_confirm" placeholder="Confirm Password" required>
     <input type="submit" name="submit_button" value="Sign Up">
     <a href="/login" target="_blank">Already registered? Goto Login Page.</a>
+</form> -->
+<div class="blur">
+    <div class="top-text">
+<h1 class="top-text-h1" > < FD /> Forge Dth </h1>
+<span class="btn-login"><a href="/login"> Login </a></span>
+</div>
+</div>
+<div class="main-body">
+<div class="main-body-text-1">
+
+</div>
+<div class="main-body-text">
+<div class="login-card" style="height: 450px;">
+  <h1>
+      Sign up
+  </h1>
+  <hr/>
+  <form class="container box" action="" method="POST" class="form-box">
+  <input placeholder="Email" class="card-input btn-login-padding" type="email">                  
+</input>
+<input placeholder="Username" class="card-input btn-login-padding " type="text">                  
+</input>
+<input placeholder="Password" class="card-input btn-login-padding" type="password">                  
+</input>
+<input placeholder="Password" class="card-input btn-login-padding" type="password">                  
+</input>
+<br/>
+ <br/> <a href="/login">already have an account? login </a>
+
+<br/>
+<br/>
+<input class="btn-long"  type="submit" name="submit_button" value="Sign Up">    </input> 
 </form>
+</div>
+</div>
+</div>
```

Comment:
Please also run a html linter, so that code is easily readable and is standaradized.

Reply:
sure I will do that